src/main/resources/assets/sodium/lang/ru_ru.json

```
@@ -0,0 +1,66 @@
+  "sodium.options.animate_only_visible_textures.name": "Анимировать только видимое",
+  "sodium.options.animate_only_visible_textures.tooltip": "Если включено, будут обновляться лишь видимые анимированные текстуры. Это может значительно повысить частоту кадров на некоторых устройствах, особенно с \"тяжёлыми\" пакетами ресурсов. Если у каких-то текстур возникают проблемы с анимацией, попробуйте отключить эту опцию.",
+  "sodium.options.cpu_render_ahead_limit.name": "Лимит пре-рендеринга",
+  "sodium.options.cpu_render_ahead_limit.tooltip": "Задаёт максимальное количество кадров, которые ЦП может подготавливать для рендеринга на ГП. Очень низкие или высокие значения могут дестабилизировать частоту кадров.",
+  "sodium.options.cpu_render_ahead_limit.value": "%s кадр(ов)",
+  "sodium.options.allow_direct_memory_access.name": "Разрешить прямой доступ к памяти",
+  "sodium.options.allow_direct_memory_access.tooltip": "Если включено, некоторым критическим путям кода будет разрешено использовать прямой доступ к памяти для повышения производительности. Зачастую это сильно снижает перегрузку ЦП при рендеринге чанков и сущностей, но может усложнить диагностику некоторых ошибок и вылетов. Отключать это стоит только если вы следуете инструкциям или точно знаете, что делаете.",
+  "sodium.options.enable_memory_tracing.name": "Включить трассировку памяти",
+  "sodium.options.enable_memory_tracing.tooltip": "Функция отладки. Если включено, трассировка стека будет собираться вместе с распределением памяти, чтобы помочь улучшить диагностическую информацию при обнаружении утечек памяти.",
+  "sodium.options.performance_impact_string": "Влияние на производительность: %s",
+  "sodium.options.use_persistent_mapping.name": "Использовать постоянный буфер",
+  "sodium.options.use_persistent_mapping.tooltip": "Если включено, небольшое количество памяти будет постоянно использоваться в качестве промежуточного буфера для загрузки чанков, помогая уменьшить перегрузку ЦП и нестабильность времени отрисовки кадра при загрузке/выгрузке чанков.\n\nТребуется OpenGL 4.4 или ARB_buffer_storage.",
+  "sodium.options.chunk_memory_allocator.name": "Распределитель памяти",
+  "sodium.options.chunk_memory_allocator.tooltip": "Выбирает распределитель памяти, используемый для рендеринга чанков.\n- ASYNC: Быстрейший вариант, хорошо работает с большинством современных графических драйверов.\n- SWAP: Запасной вариант для старых графических драйверов. Может существенно увеличить использование памяти.",
+  "sodium.options.chunk_memory_allocator.async": "Async",
+  "sodium.options.chunk_memory_allocator.swap": "Swap",
+  "sodium.options.chunk_update_threads.name": "Потоки обновления чанков",
+  "sodium.options.chunk_update_threads.tooltip": "Задаёт количество потоков для сборки чанков. Использование большего количества потоков способствует повышению скорости обновления и загрузки чанков, но может негативно повлиять на время отрисовки.",
```

Судя по устоявшейся терминологии, "Frame time" следует переводить как "время кадра".
Как по мне, предыдущий вариант был лучше из-за бо́льшей понятности для новичков, но думаю, что так будет правильнее.

```suggestion
  "sodium.options.chunk_update_threads.tooltip": "Задаёт количество потоков для сборки чанков. Использование большего количества потоков способствует повышению скорости обновления и загрузки чанков, но может негативно повлиять на время кадра.",
```